packit/jobs.py

```diff
@@ -298,6 +298,19 @@ def run(self):
         )
 
 
+# @add_to_mapping
```

Comment:
will we do fedmsg or webhooks here?

Reply:
fedmsg - As we talked before, I was struggling to find info about webhooks triggered by copr. Could you point me to some if you know of something?

Reply:
I don't to be honest, I'd ask someone on the copr team, especially since they pointed you to the v3 API.